app/src/main/res/drawable/ic_palette.xml

```diff
@@ -0,0 +1,10 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
Copyright is missing and file should be added in NOTICE.txt (#168 will be our savior 🙏)